migration/20221014150625-modify-student-table.js

```diff
@@ -0,0 +1,13 @@
+const { MigrationInterface, QueryRunner } = require("typeorm");
+
+module.exports = class modifyStudentTable20221014150625 {
+
+    async up(queryRunner) {
+        await queryRunner.query(
+            `alter table \`student\` add \`tr_name\` varchar(100) not null,
+            add \`tr_surname\` varchar(100) not null,
+            add \`gender\` enum('Male','Female') not null ;`,);
+
+    }
```

Comment:
You should the down method also with a script to remove the added columns.
@RamaMahjoub